developer-docs-site/docs/nodes/full-node/run-a-fullnode-on-gcp.md

```diff
@@ -93,7 +93,13 @@ You can deploy a public fullnode on GCP by using the Aptos fullnode Terraform mo
   touch main.tf
   ```
 
-4. Modify `main.tf` file to configure Terraform, and create public fullnode from Terraform module. Example content for `main.tf`:
+4. Modify the `main.tf` file to configure Terraform and create a public fullnode from the Terraform module.
+
+**Note:** If you are using a different version of Terraform, you will need to use the `tfenv` command to change the required version. 
```

Comment:
This is assuming that they have `tfenv` installed, but I think that's fine for now / I don't remember if we tell you to install it.